terraform/modules/benchmark/service/benchmark_base.rb

```diff
@@ -58,19 +80,26 @@ def proc_monitor(pid, benchmark)
   end
 
   # Helper function for measuring execution time; passes process ID to online
-  # validation and process monitor.
+  # validation and process monitor and Measure Peak Memory.
   def exec_command_aux(command, metrics, benchmark)
-    Open3.popen2(*command) do |i, o, t|
+    command_with_time = ['/usr/bin/time', '-v'] + command
```

Comment:
won't `['/usr/bin/time', '-v', command]` work?

Reply:
why?

Reply:
Do you know why?

Reply:
So how can you fix this?

Reply:
I'd rather have something more explicit (the `command` being array is a bit unexpected), but I'm okay with leaving this as is.